### DISCUSSION TITLE
profile README.md doesn't display on profile page

### DISCUSSION THREAD

**Discussion by Jkhall81** [community/community#69234] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Like a lot of people I have a profile README.md file to display information about myself when people visit my github profile page.  For some reason it isn't working.  It used to work fine.

The repo with said README.md file has the same name as my account name, and is set to public.  The README.md contains content.  When I noticed that the README wasn't displaying I also noticed that a code stats badge, languages used percentage, also isn't working.  It has an error message from Vercel about something related to my username being an organization or something.

I'm looking for a job, and would like to get my profile up and running ASAP.  I tried searching stack overflow and google, but can't figure out why my profile README is not displaying.  Thank you in advance.

**Comment by yttasdfghjk** [comment]
im facing the same issues, the readme used t be displayed in the profile section but this stopped suddenly. how did you fix it?

**Comment by jadebluestar** [comment]
i was facing the same issue because I had renamed my readme repo, but it usually works once you change the erepo name back to your username

**Comment by 07Codex07** [comment]
Any updates? Now I am facing the same issue??

**Comment by kevinwangombe** [comment]
Hi! Here's how to fix your profile README not displaying:
The repo must match your username exactly — if your GitHub username is jkhall81, the repo must be named jkhall81 (case sensitive). Double check this.
Make sure the repo is set to Public — go to the repo Settings → scroll down → change visibility to Public if it isn't already.
The README.md must be in the root of the repo — not inside any folder, just directly in the repo.
For the Vercel/stats badge error — this usually means the github-readme-stats instance can't find your account. Try replacing your stats URL with:
https://github-readme-stats.vercel.app/api?username=YOUR_USERNAME

After fixing the repo name and visibility, it should display within a few minutes.

**Comment by aminehd** [comment]
I have the same issue https://github.com/aminehd

**Reply by rababab** [reply]
hey it seems like it's working now for you can i please know how you fixed it, 
https://github.com/rababab

edit: nvm i fixed it, just got a share to profile button and clicked i and that worked
